app/test_main.py

```diff
@@ -1,3 +1,39 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, goals",
+    [pytest.param(14, 14, [0, 0],
+                  id="should return first 15 dog and cat years"),
```

Comment:
The expected result `[0, 0]` for `cat_age=14` and `dog_age=14` is correct according to the task description, as both ages are less than 15 years, which should result in 0 human years for both.